cli/lakeflags/flags.go

```diff
@@ -100,7 +101,7 @@ func (l *Flags) URI() (*storage.URI, error) {
 	if l.Lake == "" {
 		return nil, errors.New("lake location must be set (either with the -lake flag or ZED_LAKE environment variable)")
 	}
-	u, err := storage.ParseURI(l.Lake)
+	u, err := storage.ParseURI(strings.TrimRight(l.Lake, "/"))
```

Comment:
Instead of doing this here, I think we should do it in api/client.NewConnectionTo so anyone using that package benefits.

If you agree, let's also make the equivalent change in python/zed/zed.py.
```diff
     def __init__(self,
                  base_url=os.environ.get('ZED_LAKE', 'http://localhost:9867'),
                  config_dir=os.path.expanduser('~/.zed')):
-        self.base_url = base_url
+        self.base_url = base_url.rstrip('/')
         self.session = requests.Session()
         self.session.headers.update({'Accept': 'application/x-zjson'})
         token = self.__get_auth_token(config_dir)
```

Reply:
Good idea 👍🏻